### PR TITLE
RA-429 : deleting duplicate concepts

### PR DESCRIPTION
**Pull request by mozzy11**
https://issues.openmrs.org/browse/RA-429

added liquibase file that deletes duplicate concepts

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.referencemetadata;
+
+import java.util.Collection;
+
+import org.openmrs.Concept;
+import org.openmrs.api.context.Context;
+
+public class AdministrativeNotification {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: Has this class got anything to do with the ticket?

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: well i was just trying to figure out , how i can display an administrative notification on the user UI .. but i failed to get exactly an example of how to use the  org.openmrs.module.appframework.factory.AdministrativeNotificationProduce

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: How is it related to this ticket?

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: i thought that method can be used to check whether duplicate concepts exists , and then trigers the administration Notification requested by he ticket

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: though am not yet sure how to display an administration notification on the UI as done in the atlas module. i failed to gt any reference documentation on how to do it

Comment (samuelmale) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: I'm sure this class is redundant. What is that inevitable usecase for introducing this class?

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: oh ya samwel , acoording to the acceptance criteria  ...
*Upon upgrading an existing OpenMRS installation, if it has two TRUE or FALSE concepts, the administrator should be shown instructions about how to clean this up in their database.*

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: soi was trying out making a method that will trigger the amin notification

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: but am soon pushing , i have modified that

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: hi dan , let me close this PR, i have created a new one

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java]: please look at this   https://github.com/openmrs/openmrs-module-referencemetadata/pull/40

and https://github.com/openmrs/openmrs-module-referenceapplication/pull/50

am closing this

**Pull request closed without merging**